requirements.txt

```
@@ -23,7 +23,7 @@ toml==0.10.0
 requests==2.21.0
 
 # Api dependencies
-djangorestframework==3.9.0
+djangorestframework==3.9.1
```

```suggestion
djangorestframework==3.9.4
```

Autant aller à la dernière version, non ? :)

---

pas faux j'avais mis la .1 car c'était elle qui corrigeait la vulnérabilité